.gitattributes

```diff
@@ -9,3 +9,8 @@ src/etc/installer/gfx/* binary
 *.woff binary
 src/vendor/** -text
 Cargo.lock -merge linguist-generated=false
+
+# Older git versions try to fix line endings on images, this prevents it.
+[attr]img eol=auto
+*.png img
+*.ico img
```

Comment:
Why not use `*.png binary` instead?

Reply:
Edit: I find this [useful repo](https://github.com/alexkaratarakis/gitattributes/blob/master/Web.gitattributes).

Reply:
Good point!